backend/src/main/java/com/htecgroup/skynest/controller/UserController.java

```diff
@@ -258,4 +259,13 @@ public ResponseEntity<String> deleteUser(@PathVariable UUID uuid) {
     log.info(deleteSuccess);
     return ResponseEntity.ok(deleteSuccess);
   }
+
+  @PostMapping("/password-change/{uuid}")
+  public ResponseEntity<Boolean> changePassword(
+      @Valid @RequestBody UserChangePasswordRequest userChangePasswordRequest,
+      @PathVariable UUID uuid) {
+    userService.authorizeAccessToUserDetailsWith(uuid);
```

Comment:
Should an admin have the power to change other people's passwords? I think this way he can, just wondering if that's correct

Reply:
maybe you need to ask again. It is really strange to give somebody power to change your password.
Honestly if that is real app behavior , I would never use that app :)

Reply:
Any user, admin or otherwise can change their own password. We'll leave it like that and when we can ask the client how it should be, we'll change it if we have to.